Gemfile.lock

```diff
@@ -467,7 +467,7 @@ GEM
       twitter-text (~> 1.4)
     sitemap_generator (5.1.0)
       builder
-    social-share-button (0.1.8)
+    social-share-button (0.1.10)
```

Comment:
Did you mean to update `social-share-button`? It's probably not a problem, but...